### DISCUSSION TITLE
can't force match to eof

### DISCUSSION THREAD

**Discussion by cppden** [taocpp/PEGTL#42] [Q&A]
Don't understand why this doesn't work as one would expect - match till eof or fail?
```cpp
struct grammar : pegtl::must< pegtl::one<'a'>, pegtl::star< pegtl::seq< pegtl::one<','>, pegtl::one<'a'> >, pegtl::eof > > {};

char const csz[] = "a,b";
if (!pegtl::parse_memory< grammar >(csz, csz+ sizeof(csz)-1, "csz")) {
  std::cout << "FAILED!" << std::endl;
} else {
  std::cout << "passed." << std::endl;
}
```

**Comment by cppden** [comment]
I'm sorry. My fault a typo in last closing bracket. :(
Guys do you have any mailing list or something to ask questions?
Thanks!

**Comment by ColinH** [comment]
Glad you got it sorted out! We don't have a mailing list yet...
